package-lock.json

```diff
@@ -39312,7 +39312,7 @@
             "dev": true,
             "requires": {
                 "exec-sh": "^0.3.2",
-                "minimist": "^1.2.0"
```

Comment:
why is there a package_lock.json changes?

Reply:
I did npm install to install @msnyder-msft changes, and this came out

Reply:
did you do `npm ci`?

Reply:
resolved with npm ci --legacy-peer-deps